### PR TITLE
IDENTITY-4047 Create Fine Grained Permissions for Workflow feature

### PR DESCRIPTION
**Pull request by cdwijayarathna**

### CODE REVIEW

**Comment by cdwijayarathna** [comment]
Depends on https://github.com/wso2/carbon-identity/pull/1383

**Comment by cdwijayarathna** [comment]
Need to merge https://github.com/wso2/product-is/pull/318 as well with this.

File: components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt.ui/src/main/resources/web/workflow-mgt/wf-request-list.jsp [hunk 1]
```diff
@@ -445,8 +445,9 @@
                            onclick="listWorkflows('<%=workflowReq.getRequestId()%>');return false;"
                            href="#" style="background-image: url(images/list.png);"
                            class="icon-link"><fmt:message key='workflows'/></a>
-                        <% if (workflowReq.getCreatedBy() != null && workflowReq.getStatus().equals("PENDING") &&
-                                workflowReq.getCreatedBy().equals(loggedUser)) { %>
+                        <% if ( workflowReq.getStatus().equals("PENDING") && CarbonUIUtil.isUserAuthorized(request,
```
Comment (pulasthi7) [inline components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt.ui/src/main/resources/web/workflow-mgt/wf-request-list.jsp]: use CONSTANT.equals(variable) pattern when comparing strings

Comment (cdwijayarathna) [inline components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt.ui/src/main/resources/web/workflow-mgt/wf-request-list.jsp]: Thanks, Fixed with https://github.com/wso2/carbon-identity/pull/1675

**Pull request merged**